config/example-extras.cfg

```diff
@@ -583,6 +583,38 @@
 #a0_pin
 #   The pins connected to an uc1701 type lcd. These parameters must be
 #   provided when using an uc1701 display.
+#menu_root:
+#   Entry point for menu, root menuitem name. This parameter must be
+#   provided when using menu.
+#menu_timeout:
+#   Timeout for menu. Being inactive this amount of seconds will trigger
+#   menu exit or return to root menu when having autorun enabled [optional]
+#menu_autorun:
+#   Enable this to run menu immediately at startup. It's mostly used to replace
+#   stock info screens with custom ones. It accepts boolean values,
+#   default is False. [optional]
+#rows:
+#   Only needed when using display with custom row count [Optional].
+#cols:
+#   Only needed when using display with custom column count [Optional].
```

Comment:
Very minor, but I'd prefer to not add rows/cols fields to the menu.  I understand some displays can be configured to use different rows, but the low-level lcd_chip display code would also need to change for that to work.  So, I fear adding a config option that doesn't actually do what a user thinks it will do will just lead to confusion.  When we're ready to support rows/cols and have tested it on real hardware, then add config support for it.